docs/en/sql-reference/statements/insert-into.md

```diff
@@ -204,6 +204,20 @@ Result:
 └─────┴───────────────────────┘
 ```
 
+## Inserts into ClickHouse Cloud
+
+By default, services on ClickHouse Cloud provide multiple replicas for high availability. When you connect to a service, a connection is established to one of these replicas.
+
+After an `INSERT` succeeds, data is written to the underlying storage. However, it may take some time for replicas to receive these updates. Therefore, if you use a different connection that executes a `SELECT` query on one of these other replicas, the updated data may not yet be reflected.
+
+It is possible to use the `select_sequential_consistency` to force replicas to receive the latest updates. Here is an example of a SELECT query using this setting:
```

Comment:
AFAIK, `select_sequential_consistency` doesn't force all replicas to get updates, just the one you are reading from and only for the involved tables.